addons/account/models/account_payment.py

```diff
@@ -659,6 +663,24 @@ def _compute_payment_receipt_title(self):
         """ To override in order to change the title displayed on the payment receipt report """
         self.payment_receipt_title = _('Payment Receipt')
 
+    @api.depends('partner_id', 'amount', 'date')
+    def _compute_duplicated_payment_ids(self):
+        """ Retrieve ids of payments with same partner_id, amount and date as the current payment """
+        for payment in self:
+            payment.has_duplicated_payments = 0
+            payment.duplicated_payment_ids = None
```

Comment:
Had to add this because some tests were failing with ValueError, for example: Compute method failed to assign account.payment(2,).duplicated_payment_ids
This was the solution I found, but maybe there is a more elegant way to do it

Reply:
It's fine by me, i usually see this in a else and not at the beginning a the function, i suppose it can be better in terms of assignation of variable since here you have so assign it twice if it go though the if statement but then it can be a bit less readable, maybe you can try the other version and show me and we can decided which one is better :smile: